add new map info format; pump package version

pqhuy98 opened the pull request:
Taken from https://github.com/cipherxof/mdx-m3-viewer/commits/updates

flowtsohg commented:
I would have preferred less automatic style changes but whatever.
Please remove the package lock file and file saver dependency (technically the package lock should be commited, but for newer code of mine I opted to do the "more" correct thing and simply lock all the package versions, maybe I'll do it here as well).
Other than that looks good, thanks. :)

pqhuy98 committed:
add new map info format; pump package version

pqhuy98 commented:
I removed package-lock.json and also added it to .gitignore.

About file-saver, I see it is used in https://github.com/flowtsohg/mdx-m3-viewer/blob/9c0306f9043729d6a63367445b88d099371b18ed/clients/mdlx/index.js#L4 but not included in package.json. I got an error missing module when open that script.

flowtsohg committed:
Use the internal file-saver file in the mdlx client

flowtsohg commented:
True about file saver. It looks like I messed up something at some point, since the clients were meant to use the internal copy of it in the thirdparty folder.
Changed it to use the internal one.

Thanks for this update! :)
